client/src/components/Cart-item/cart-item.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
displays summary of the items added by the user in the `cart dropdown` component